src/Components/Button/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+function Button({buttonType, buttonText, onClickFunction}) {
+    return (
+        <button
+            type={buttonType}
+            onClick={onClickFunction}
+        >
+            {buttonText}
+        </button>
+    );
+}
```

Comment:
De aanroep van dit component kost net zoveel regels code dan direct het `button` component gebruiken. Daarom is dit zelfgemaakte component eigenlijk wat overbodig